recipe/meta.yaml

```diff
@@ -8,37 +8,50 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 46d3fd2016cd5b842283d547877a02c418a0fe9aa7a6b0ae344115a2c990fef4
+  patches:
+    - patches/0001-rm-test-jsonpath-py.patch
+    - patches/0002-safe-eval.patch
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
 
 test:
+  source_files:
+    - test
   imports:
     - jsonpath
   requires:
+    - pip
     - pytest
-  source_files:
-    - test
   commands:
-    - pytest test/test1.py test/test2.py test/test3.py
+    - pip check
+    - pytest -v test/test1.py test/test2.py test/test3.py test/test4.py
 
 about:
-  home: https://pypi.org/project/jsonpath
+  home: https://www.ultimate.com/phil/python/#jsonpath
   license: MIT
   license_family: MIT
+  # conda-forge created LICENSE.txt but the actual license statement
+  # is in jsonpath.py
```

Comment:
This is a little worrying. Isn't it like an unspoken rule that licenses be in their own file?

Reply:
Hah, this has come up twice in the last day!
IIUC, a license (and copyright!) applies to a file.  Some files may choose to indicate that another file contains the actual licence wording (this is particularly common where the license wording is more than a couple of paragraphs -- "... see LICENSE").
Cue philosophical question: Is a LICENSE file sat in the top of a tree with no-one referring to it "valid"?

Reply:
```suggestion
  # The actual license statement is in jsonpath.py which is also the only source code file
  # in the repository. We used the same approach of conda-forge and we extracted the 
  # license from the header and placed it in the file recipe/LICENSE.txt.
```

Reply:
we agreed with SF that this approach is fine

Reply:
"We used the same approach of conda-forge and we extracted the license from the header and placed it in the file recipe/LICENSE.txt."

We didn't, we just forked the conda-forge repo.  I was only commenting on the fact that `LICENSE.txt` is not in the upstream repository.

Reply:
> we just forked the conda-forge repo

we used their approach ^^

they also have extracted the license header into a file

anyway, just a suggestion